packages/contracts-bedrock/contracts/test/invariants/L2OutputOracle.t.sol

```diff
@@ -0,0 +1,89 @@
+pragma solidity 0.8.15;
+
+import { L2OutputOracle } from "../../L1/L2OutputOracle.sol";
+import { L2ToL1MessagePasser } from "../../L2/L2ToL1MessagePasser.sol";
+import { Predeploys } from "../../libraries/Predeploys.sol";
+import { Test } from "forge-std/Test.sol";
+
+contract L2OutputOracle_Invariants is Test {
+    // Global
+    address multisig = address(512);
+
+    // Test target
+    L2OutputOracle oracle;
+
+    L2ToL1MessagePasser messagePasser =
+        L2ToL1MessagePasser(payable(Predeploys.L2_TO_L1_MESSAGE_PASSER));
+
+    // Constructor arguments
+    address internal proposer = 0x000000000000000000000000000000000000AbBa;
```

Comment:
These could technically all be constants

Reply:
This was a direct copy of the `L2OutputOracle_Initializer` without the `CommonTest` inheritance- before we could target a specific contract (`InvariantTest.sol` in forge-std), it would run forever due to trying to hit the `FFIInterface` contract's functions. Should be able to remove this duplicated boilerplate, but can update the one in `CommonTest` 😄